docs/Changelog.md

```diff
@@ -4,6 +4,104 @@ All notable changes to this project will be documented in this file.
+* Fix --info and --help when tools are registered by @Evangelink in [#3123](https://github.com/microsoft/testfx/pull/3123)
+* Fix --info when tools are present by @Evangelink in [#3118](https://github.com/microsoft/testfx/pull/3118)
+* Fix MSTEST0029 by @engyebrahim in [#3090](https://github.com/microsoft/testfx/pull/3090)
+* Fix assembly resolution with DeploymentItem (#3034) by @XXX in [#3036](https://github.com/microsoft/testfx/pull/3036)
+* Fix assembly resolution error by @XXX in [#2948](https://github.com/microsoft/testfx/pull/2948)
+* Fix property name for enabling aspire in MSTest.Sdk by @XXX in [#2888](https://github.com/microsoft/testfx/pull/2888)
+* Fix MSTEST0014 FP with arrays by @XXX in [#2857](https://github.com/microsoft/testfx/pull/2857)
+* Workaround the UWP release mode issue with DataRow data by @XXX in [#3240](https://github.com/microsoft/testfx/pull/3240)
+* Handle InvalidOperationException when accessing process ID by @XXX in [#3250](https://github.com/microsoft/testfx/pull/3250)
+* MSTEST0004: Report only on non-abstract, non-static, classes by @XXX in [#3230](https://github.com/microsoft/testfx/pull/3230)
+* Downgrade some log levels from info to debug by @Evangelink in [#3206](https://github.com/microsoft/testfx/pull/3206)
+* Retrieve PID inside try/catch by @Evangelink in [#3193](https://github.com/microsoft/testfx/pull/3193)
+* Shorten pipe names by @Evangelink in [#3183](https://github.com/microsoft/testfx/pull/3183)
+* dispose of IProcess by @SimonCropp in [#3141](https://github.com/microsoft/testfx/pull/3141)
+* ClassCleanup are not executed if testclass is ignored by @Evangelink in [#3142](https://github.com/microsoft/testfx/pull/3142)
+* Ensure ValueTask test methods are awaited by @Evangelink in [#3137](https://github.com/microsoft/testfx/pull/3137)
+* Print exceptions occurring during Task.WaitAll by @Evangelink in [#3128](https://github.com/microsoft/testfx/pull/3128)
+* Discriminate MissingMethodException from hot reload by @Evangelink in [#3098](https://github.com/microsoft/testfx/pull/3098)
+* Check that Reflect Helper caches inherited and non-inherited attributes separately by @nohwnd in [#3117](https://github.com/microsoft/testfx/pull/3117)
+* Fixes displaying arity for many (N instead of int.Max) by @engyebrahim in [#3115](https://github.com/microsoft/testfx/pull/3115)
+
+### Artifacts
+
+* MSTest: [3.5.0](https://www.nuget.org/packages/MSTest/3.5.0)
+* MSTest.TestFramework: [3.5.0](https://www.nuget.org/packages/MSTest.TestFramework/3.5.0)
+* MSTest.TestAdapter: [3.5.0](https://www.nuget.org/packages/MSTest.TestAdapter/3.5.0)
+* MSTest.Analyzers: [3.5.0](https://www.nuget.org/packages/MSTest.Analyzers/3.5.0)
+* MSTest.Sdk: [3.5.0](https://www.nuget.org/packages/MSTest.Sdk/3.5.0)
+* Microsoft.Testing.Extensions.CrashDump: [1.3.0](https://www.nuget.org/packages/Microsoft.Testing.Extensions.CrashDump/1.3.0)
```

Comment:
Intended to list these here?

Reply:
I have started to do that on the previous release to make it "easy" to find the extensions available for "MSTest runner" but I am fine to change that.